api/shape/get.js

```diff
@@ -27,10 +29,12 @@ export const get = Shape.chainable((...tags) => (shape) => {
   };
   const geometry = shape.toGeometry();
   visit(geometry, walk);
-  return Group(...picks);
+  return groupOp(...picks);
```

Comment:
What is groupOp?

Reply:
Nevermind...I see where it's defined. I thought it might be a global thing

Reply:
It's a function for grouping the results.

Usually it is Group, but I needed to be able to produce a list sometimes.

get('foo', List)